index.html

```diff
@@ -80,121 +86,88 @@ <h1>I’m Beyk
 					</div>
 				</div>
 			</section>
-			<!--Start work section-->
-			<section class="work" id="work">
-				<div class="work-tonic">
-					<div class="snapshoot-div">
-						<img src="images/Snapshoot1.png" alt="Project image" class="snapshoot"/>
-						<img src="images/snapshoot1-des.png" alt="Project image" class="desk-snapshoot"/>
-					</div>
-					<div class="work-info">
-						<h2>Tonik</h2>
-						<div class="text-info">
-							<ul>
-								<li class="client-name">CANOPY</li>
-								<li class="role">Back End Dev</li>
-								<li class="year">2015</li>
-							</ul>
-						</div>
-						<p class="work-text"
-							>A daily selection of privately personalized reads; no accounts or sign-ups required.</p
-						>
-						
-							<ul class="tags">
-								<li>html</li>
-								<li>css</li>
-								<li>javaScript</li>
-							   </ul>
-						
-						<button class="see-proj">See project</button>
-					</div>
-				</div>
 			
-				
-				<div class="work-tonic reverse">
-					<div class="snapshoot-div">
-						<img src="images/Snapshoot2.png" alt="Project image" class="snapshoot"/>
-						<img src="images/snapshoot2-des.png" alt="Project image" class="desk-snapshoot"/>
+			<section class="detail-section">
```

Comment:
- [ ] In this particular project, the whole projects section along with the modals should be created dynamically with Javascript.

The base projects section is already done (good job!), but the modals are still not being created with Javascript.  Kindly please use Javascript to dynamically render your modals.

You could reuse the code that you already have.

That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `data` array.

Reply:
Hello man, Which section do you mean? detail section?

Reply:
Project requirements:  You need to update the **main page** so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

Reply:
Do I need to make a dynamic Popup section?

Reply:
Give me a minute I'm reviewing this inquiry.

Reply:
Yes, modal popup data needs to be populated with the data from the `projects` array (in your case that array is named `data`.

This is so when you add another project, you will only need to add another element to the `data` array.

Reply:
Ok, Thank you so much. I'll fix it

Reply:
I need more help. when I should dynamically call both of them it doesn't work. and I check other ropo's of my partners, I couldn't find any complete project as you want, Would you give me a hand

Reply:
You could just use the same info on the `data` object to fill both the project and the popup.

Reply:
Also, you could put some id on all the projects, and make sure the id of each project matches the id on the `data` array.  Then, add an event listener to each button, so that the correct project id is shown.  (These are just some ideas hope it's helpful).